bluehost.com.hosting.json

```diff
@@ -0,0 +1,167 @@
+{
+  "providerId": "bluehost.com",
+  "providerName": "Bluehost",
+  "serviceId": "hosting",
+  "serviceName": "hosting",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.bluehost.com",
+  "syncRedirectDomain": "my.bluehost.com",
+  "logoUrl": "https://bluehost-cdn.com/media/app/bluehost/live/static/media/src/common/assets/images/brand/bh/logo.svg",
+  "description": "Enables a domain to work with Bluehost hosting",
+  "variableDescription": "ip is the IP address of the server, and box is the hostname of the server.",
+  "records": [
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "@",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "whm",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "mail",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "cpcalendars",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "@",
+      "type": "CNAME",
+      "host": "www",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "@",
+      "type": "CNAME",
+      "host": "ftp",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "@",
+      "type": "CNAME",
+      "host": "cpanel",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "@",
+      "type": "CNAME",
+      "host": "webmail",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "mail.%domain%",
+      "type": "CNAME",
+      "host": "imap",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "mail.%domain%",
+      "type": "MX",
+      "host": "@",
+      "ttl": 14400,
+      "priority": 0
+    },
+    {
+      "data": "v=spf1 a mx include:websitewelcome.com ~all",
+      "type": "TXT",
+      "host": "@",
+      "ttl": 14400,
+      "txtConflictMatchingMode": "Prefix",
+      "txtConflictMatchingPrefix": "v=spf1"
```

Comment:
We prefer to use the SPFM convention for SPF records.  See the spec.

Reply:
OK. Will look that up and make the appropriate changes.

I had originally done this 4 months ago to work with Google and we're just now getting it contributed here.

Reply:
Change made in a04edc5.